### PR TITLE
Link connection to token user while importing a mapping

### PR DESCRIPTION
**Pull request by codingjoe**

### CODE REVIEW

File: heroku_connect/management/commands/import_mappings.py [hunk 1]
```diff
@@ -20,27 +20,43 @@ def add_arguments(self, parser):
...
-            try:
-                connections = utils.get_connections(app_name)
-            except requests.HTTPError as e:
-                raise CommandError("Failed to load connections") from e
+            connections = self.get_connections()
             if len(connections) == 0:
-                raise CommandError("There are no connections associated "
-                                   "with the app '%s'." % app_name)
+                msg = self.style.NOTICE(
+                    "There is no connection associated with current user the app '%s'."
```
Comment (tilboerner) [inline heroku_connect/management/commands/import_mappings.py]: Better: 

* "There is no connection associated with the app %r for the current user."
* "No associated connections found for the current user with the app %r."

File: heroku_connect/management/commands/import_mappings.py [hunk 2]
```diff
@@ -20,27 +20,43 @@ def add_arguments(self, parser):
...
+                    " Trying to link the current user with Heroku Connect..." % self.app_name
```
Comment (tilboerner) [inline heroku_connect/management/commands/import_mappings.py]: Do we owe the user a success message after "Trying ..."? To be able to remain silent on success, I think it's better to be affirmative and drop the ellipsis: 

* "Linking the current user with Heroku Connect.", or 
* "Ensuring that the current user is linked with Heroku Connect."

File: heroku_connect/management/commands/import_mappings.py [hunk 3]
```diff
@@ -20,27 +20,43 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         connection_id = options.get('CONNECTION_ID', None)
-        app_name = options.get('HEROKU_APP', settings.HEROKU_CONNECT_APP_NAME)
-        if not (connection_id or app_name):
+        self.app_name = options.get('HEROKU_APP', settings.HEROKU_CONNECT_APP_NAME)
```
[the remark this answers is not in the document]
Comment (codingjoe) [reply heroku_connect/management/commands/import_mappings.py]: Interesting thought. I agree that is the more beautiful design. I'd really like to explain that thought to the rest of the team on Friday.

[the remark this answers is not in the document]
Comment (tilboerner) [reply heroku_connect/management/commands/import_mappings.py]: Fine with me. I'd be happy to do that as well, it's a good excuse to finally get my laptop set up for mobile use and chrome-casting. :grin:

File: heroku_connect/management/commands/import_mappings.py (hunk 2 above)
Comment (codingjoe) [reply heroku_connect/management/commands/import_mappings.py]: True, I agree those messages should state a clean intent.

**Pull request merged**